### PR TITLE
Izzy the Iguana, master of Pipes

### PR DESCRIPTION
**Pull request by swifterjack**
## About The Pull Request
Adds an iguana. Sprites by Torimaesu.

iguana_forward

## Why It's Good For The Game
Dude hell yeah iguana pet for the gec!

iganga
## Changelog
:cl:
add: Iguana
/:cl:

### CODE REVIEW

**Comment by Apogee-dev** [comment]
don't forget to make iguana whip it good instead of bite

**Comment by swifterjack** [comment]
> don't forget to make iguana whip it good instead of bite

they can now whip. just needs a dead sprite, and this should be good

**Comment by swifterjack** [comment]
this pr is finished, should be good to merge

**Comment by swifterjack** [comment]
how do i undraft it

**Comment by rye-rice** [comment]
click the ready for review button

**Comment by Ms-Mee** [comment]
> how do i undraft it

How to charge batong?

**Comment by swifterjack** [comment]
> > how do i undraft it
> 
> How to charge batong?

how do i report you to god for your miscreant deeds

**Comment by Ms-Mee** [comment]
> > > how do i undraft it
> > 
> > How to charge batong?
> 
> how do i report you to god for your miscreant deeds

I answer only to the primal slime's thoughts and my girlfriend, everything else has no management power over mine

**Comment by swifterjack** [comment]
> > > > how do i undraft it
> > > 
> > > 
> > > How to charge batong?
> > 
> > 
> > how do i report you to god for your miscreant deeds
> 
> I answer only to the primal slime's thoughts and my girlfriend, everything else has no management power over mine

how do i report you to your girlfriend for your miscreant deeds

**Comment by Ms-Mee** [comment]
> > > > > how do i undraft it
> > > > 
> > > > 
> > > > How to charge batong?
> > > 
> > > 
> > > how do i report you to god for your miscreant deeds
> > 
> > I answer only to the primal slime's thoughts and my girlfriend, everything else has no management power over mine
> 
> how do i report you to your girlfriend for your miscreant deeds

I mean if you find out who she is she'd be more worried about some random person talking about her girlfriend than whatever thing I did

File: whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm [hunk 1]
```diff
@@ -24,3 +24,23 @@
 	speed = 8
 	glide_size = 2
 	footstep_type = FOOTSTEP_MOB_CLAW
+//shipping my tests all over the floor
```
Comment (ZephyrTFA) [inline whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm]:
```suggestion

```

File: whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm [hunk 2]
```diff
@@ -24,3 +24,22 @@
 	speed = 8
 	glide_size = 2
 	footstep_type = FOOTSTEP_MOB_CLAW
+/mob/living/simple_animal/hostile/retaliate/gator/steppy/iguana
```
Comment (ZephyrTFA) [inline whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm:28]:
```suggestion

/mob/living/simple_animal/hostile/retaliate/gator/steppy/iguana
```

**Review by ZephyrTFA — APPROVED** [review]
iguana

**Comment by swifterjack** [comment]
> iguana

iguana

**Comment by rye-rice** [comment]
Computer, activate iguana.

**Comment by TheNeoGamer42** [comment]
> Computer, activate iguana.

Computer, Deactivate iguana inhibitors, MAXIMUM IGUANA, NOW.

File: whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm [hunk 3]
```diff
@@ -24,3 +24,23 @@
 	speed = 8
 	glide_size = 2
 	footstep_type = FOOTSTEP_MOB_CLAW
+
+/mob/living/simple_animal/hostile/retaliate/gator/steppy/iguana
+	name = "Izzy"
+	desc = "A master of looking at pipes, she's a favorite among the GEC for her GECK-like nature."
```
Comment (MarkSuckerberg) [inline whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm:30]:
```suggestion
	desc = "A master of looking at pipes, she's a favorite among the GEC for her GECK-like nature."
	gender = FEMALE
```
let's keep it consistent

**Pull request merged**